platforms/sdl/base/AppPlatform_sdl_base.cpp

```diff
@@ -13,7 +13,7 @@
 
 #include "common/Utils.hpp"
 
-#include "SoundSystemAL.hpp"
+#include "platforms/openal/SoundSystemAL.hpp"
```

Comment:
Please no. Just setup the include paths correctly in the VS project! The whole point of the include path is so that the project isn't full of (annoying to change later) semi-absolute paths!

Reply:
Agreed

Reply:
I will remove this. I'm not sure why I added it, but I will make sure the VS project configured correctly to support this.

Reply:
Fixed in upcoming commit.